web_programming/get_top_billionaires.py

```diff
@@ -29,9 +29,9 @@ def calculate_age(unix_date: float) -> str:
     >>> from datetime import datetime, UTC
     >>> years_since_create = datetime.now(tz=UTC).year - 2022
     >>> int(calculate_age(-657244800000)) - years_since_create
-    73
+    72
```

Comment:
This calculation seems flawed to me.  Like it will fail around Jan 1st and then be correct again a few daze later.

Reply:
Agreed

Reply:
@cclauss Ive updated the whole function and related doctests as I dont think it was achieving what it set out to do initially. Please review changes when appropriate. I can see other PRs against this repo are failing due to this function and its doc tests currently